File: WebAPP/ClientApp/src/dto/stayEquipment.ts
import { Stay } from "./stay";

export class StayEquipment {
    public id: number;
    public stayId: number;
    public stay: Stay;
    public label: string;
    public isIncluded: boolean

    constructor(id?: number, stayId?: number, stay?: Stay, label?: string, isIncluded?: boolean)

    constructor(id: number, stayId: number, stay: Stay, label: string, isIncluded: boolean) {
        this.id = id;
        this.stayId = stayId;
        this.stay = stay;
        this.label = label;
        this.isIncluded = isIncluded;
    }
}

File: Model/StayEquipment.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Model
{
    [Table("stay_equipment")]
    public class StayEquipment
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Column("id")]
        public int? Id { set; get; }
        [Column("stay_id")]
        public int? StayId { set; get; }
        public Stay Stay { set; get; }

        [Column("label")]
        public string Label { set; get; }

        [Column("is_included")]
        public bool? IsIncluded { set; get; }
    }
}
